### ISSUE TITLE
failing on AMD Ryzen 5

### ISSUE THREAD

**Issue by kndt** [sickcodes/Docker-OSX#29]
The following when trying to run the image:

`qemu-system-x86_64: warning: dbind: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-9O0afgm3Sz: Connection refused
Could not access KVM kernel module: Permission denied
qemu-system-x86_64: failed to initialize kvm: Permission denied
`

Setup:
OS: Ubuntu 18.04
processor: AMD® Ryzen 5 3500u with radeon vega mobile gfx × 8 
graphics: AMD® Raven
arch: 64bit
RAM: 8GB
Storage: 250GB

**Comment by sickcodes** [comment]
`ls -l /dev/kvm
groups`

**Comment by sickcodes** [comment]
Check readme for troubleshooting, chmod 666 /dev/kvm should fix yours

**Comment by kndt** [comment]
Fixed, thanks.

During installation, it's showing the disk is locked. Not sure how that's possible.
